frontend/vite.config.ts

```diff
@@ -58,7 +58,7 @@ const config = ({ mode }: ConfigEnv): ReturnType<typeof defineConfig> => {
                         {
                             urlPattern: ({ url }) =>
                                 url.pathname.startsWith('/api'),
-                            handler: 'CacheFirst',
+                            handler: 'NetworkFirst',
                             options: {
```

Comment:
if you do not change this parameter, then the request to the server goes only when clearing the cache (Ctrl + Shift + R)